### ISSUE TITLE
Training and validation accuracy not changing during training

### ISSUE THREAD

**Issue by jwnsu** [brightmart/text_classification#35]
Multiple models (a02_TextCNN, a04_TextRCNN) have training and validation accuracy fixed at 0.5, while training loss/validation loss stably drops over time (dramatically, from 10^8-9 digits down to  1 to 2 digits.)

Is it normal or something wrong? 

One of models, aa6_TwoCNNTextRelation has training accuracy fluctuate (above 0.5).

Thx.

**Comment by brightmart** [comment]
yeah. it is normal, i manuall set accuracy to 0.5, you can change it in models, such as:

self.predictions = tf.argmax(self.logits, 1, name="predictions") # shape:[None,]
correct_prediction = tf.equal(tf.cast(self.predictions,tf.int32), self.input_y) #shape=[None,]
self.accuracy =tf.reduce_mean(tf.cast(correct_prediction, tf.float32), name="Accuracy") # shape=()

**Comment by brightmart** [comment]
UPDATE: if you want to try a model now, you can go to folder 'a02_TextCNN', run 'python -u p7_TextCNN_train.py', it will use sample data to train a model, and print loss and F1 score periodically.
